lib/Db/Feed.php

```diff
@@ -15,105 +15,64 @@
 
 use \OCP\AppFramework\Db\Entity;
 
-/**
- * @method integer getId()
- * @method void setId(integer $value)
- * @method string getUserId()
- * @method void setUserId(string $value)
- * @method int getOrdering()
- * @method void setOrdering(integer $value)
- * @method string getUrlHash()
- * @method void setUrlHash(string $value)
- * @method string getLocation()
- * @method void setLocation(string $value)
- * @method string getUrl()
- * @method string getTitle()
- * @method void setTitle(string $value)
- * @method string getLastModified()
- * @method void setLastModified(string $value)
- * @method string getHttpLastModified()
- * @method void setHttpLastModified(string $value)
- * @method string getHttpEtag()
- * @method void setHttpEtag(string $value)
- * @method string getFaviconLink()
- * @method void setFaviconLink(string $value)
- * @method integer getAdded()
- * @method void setAdded(integer $value)
- * @method boolean getPinned()
- * @method void setPinned(boolean $value)
- * @method integer getFolderId()
- * @method void setFolderId(integer $value)
- * @method integer getFullTextEnabled()
- * @method void setFullTextEnabled(bool $value)
- * @method integer getUnreadCount()
- * @method void setUnreadCount(integer $value)
- * @method string getLink()
- * @method boolean getPreventUpdate()
- * @method void setPreventUpdate(boolean $value)
- * @method integer getDeletedAt()
- * @method void setDeletedAt(integer $value)
- * @method integer getArticlesPerUpdate()
- * @method void setArticlesPerUpdate(integer $value)
- * @method integer getUpdateErrorCount()
- * @method void setUpdateErrorCount(integer $value)
- * @method string getLastUpdateError()
- * @method void setLastUpdateError(string $value)
- * @method string getBasicAuthUser()
- * @method void setBasicAuthUser(string $value)
- * @method string getBasicAuthPassword()
- * @method void setBasicAuthPassword(string $value)
- */
 class Feed extends Entity implements IAPI, \JsonSerializable
 {
 
     use EntityJSONSerializer;
 
-    protected $userId;
+    /** @var string */
```

Comment:
Would you mind adding a description? (I know it's not really the scope of the review but still)

Reply:
For what exactly? For the userId, every property?